app/javascript/stylesheets/basic.scss

```diff
@@ -1,49 +1,76 @@
-body {
-  background-color: #e1ebee;
-}
-
-.header {
-  background-image: url("../../../public/ivano-frankivsk2.jpg");
+.layout-body {
+  background-image: url("../../../public/vesna-frankivsk2.png");
+  height: 100vh;
+  background-position: center;
   background-repeat: no-repeat;
+  background-size: cover;
   background-attachment: fixed;
-  background-size: 100%;
-  height: 226px;
+}
+
+.layout-content {
+  background-color: rgba(255, 255, 255, 0.3);
+}
+
+.layout-first-content {
+  background-color: rgba(255, 255, 255, 0.5);
 }
 
 .navbar-brand {
+  height: 3em;
   font-weight: bold;
   font-size: x-large;
+
+  &:hover {
+    color: black;
+    font-weight: bold;
+  }
 }
 
-.auth-btn {
-  color: whitesmoke
+.auth-btn, .nav-name {
+  color: black
 }
 
 .auth-btn:hover, .btn-submit:hover, .nav-links:hover {
-  color: whitesmoke;
-  background-color: #556588;
+  color: black;
+  font-weight: bold;
 }
 
 .btn-submit {
-  color: whitesmoke;
-  background-color: #798db6;
+  color: black;
+  background-color: #3b94d9;
 }
 
 .navbar-header {
-  background-color: inherit;
+  background: none;
 }
 
 .nav-links {
-  color: white;
+  color: black;
 }
 
 #dropdownMenuButton {
   width: 158px;
 }
 
 .dropdown-lang {
-  background: #798db6;
+  background-color: rgba(255, 255, 255, 0.5);
   width: 155px;
+
+  &:hover, .layout-dropdown-item:hover {
+    font-weight: bold;
+    background-color: rgba(255, 255, 255, 0.5);
```

Comment:
rgba(255, 255, 255, 0.5) used in a lot of places

Reply:
Fixed )